src/ert/gui/tools/load_results/load_results_tool.py

```diff
@@ -14,7 +14,7 @@ def __init__(self, facade):
         )
         self.__import_widget = None
         self.__dialog = None
-        self.setEnabled(self.is_valid_run_path())
+        self.setEnabled(True)
```

Comment:
I set this to true, to allow for the test to actually get to evaluate if the runpath is valid